mozilla-lockbox/lockbox-addon
# Promotional Banners

Displays a promotional banner in the management interface when a user has an item selected:

* If the user is signed in --> "Lockbox on Mobile"
* If the user is **not** signed in --> "Sign up for FxA"

The implementation created a very basic banner widget we can extend to support closing, then specific promotional banner sub-components for device and fxa, respectively.

Fixes #14 
Fixes #79 

## Testing and Review Notes

To see the banner:
1. open Lockbox management interface
2. select an item

The banner should always be rendered.  As with other aspects of Lockbox, the implementation is responsive to state changes in Firefox; signing in (or disconnecting) will change the banner -- even while the page is still loaded.

* **not signed in**: Clicking "Sign In" takes the user to the Sync prefs page to start the signin process.
* **signed in**: Clicking "Learn More" takes the user to the Lockbox website, which will (very _very_ soon) promote the mobile apps.

## Screenshots or Videos

**not signed in**
<img width="1257" alt="image" src="https://user-images.githubusercontent.com/757401/54720105-33b64580-4b24-11e9-8e4a-19e1a3222edb.png">

**signed in**
<img width="1261" alt="image" src="https://user-images.githubusercontent.com/757401/54717946-fef3bf80-4b1e-11e9-9fa9-fe2bf21bd488.png">

## To Do
